apps/src/lib/levelbuilder/lesson-editor/LevelToken.jsx

```diff
@@ -103,24 +103,45 @@ class LevelToken extends Component {
   };
 
   handleDragStart = e => {
-    this.props.handleDragStart(this.props.level.position, e);
+    this.props.handleDragStart(this.props.scriptLevel.position, e);
   };
 
   toggleExpand = () => {
     this.props.toggleExpand(
       this.props.activityPosition,
       this.props.activitySectionPosition,
-      this.props.level.position
+      this.props.scriptLevel.position
     );
   };
 
   handleRemove = () => {
-    this.props.removeLevel(this.props.level.position);
+    this.props.removeLevel(this.props.scriptLevel.position);
+  };
+
+  scriptLevelForProgressBubble = activeLevel => {
+    let progressBubbleLevel = activeLevel;
+
+    progressBubbleLevel['isCurrentLevel'] = false;
+    progressBubbleLevel['status'] = 'not-started';
```

Comment:
is this changing from "not started" to 'not-started' and is that intentional? I'm having trouble finding a source of truth for what the valid options for this value are on the client side, and on the server I only see `not_started`.

Reply:
It looks like `LevelStatus` in shared constants is the source of truth fo the javascript code in terms of this. So I think the level should actually be marked as `LevelStatus.not_tried`